.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v2.3.0
+      hooks:
+      -   id: end-of-file-fixer
+      -   id: trailing-whitespace
+      -   id: check-yaml
+  - repo: https://gitlab.com/PyCQA/flake8
+    rev: 3.8.3
+    hooks:
+      -   id: flake8
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        args: [ --config=pyproject.toml ]
```

Comment:
I think you dont need to explicitly mention the path, it should auto detect. Just remove the args and check !